redirects-transition.conf

```diff
@@ -226,7 +241,194 @@ rewrite ^/law/cfr/ej_compilation/2007/notice_2007-2.pdf https://www.fec.gov/reso
+rewrite ^/law/cfr/ej_compilation/1983/1983-28.pdf	https://www.fec.gov/resources/cms-content/documents/notice1983-28-110283.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1983/1983-29.pdf	https://www.fec.gov/resources/cms-content/documents/notice1983-29-112183.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1983/48fr9238.pdf	https://www.fec.gov/resources/cms-content/documents/notice1983-6-030483.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1983/48fr19019.pdf	https://www.fec.gov/resources/cms-content/documents/notice1983-042783.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1983/48fr926296.pdf	https://www.fec.gov/resources/cms-content/documents/notice1983-060783.pdf redirect;
+
+# law/cfr/ej_compilation/1982/ redirects 
+rewrite ^/law/cfr/ej_compilation/1982/1982-2.pdf	https://www.fec.gov/resources/cms-content/documents/notice1982-2-040882.pdf redirect;
+
+# law/cfr/ej_compilation/1980/ redirects 
+rewrite ^/law/cfr/ej_compilation/1980/1980-8.pdf	https://www.fec.gov/resources/cms-content/documents/notice1980-8-030780.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1980/1980-14.pdf	https://www.fec.gov/resources/cms-content/documents/notice1980-14-041580.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1980/1980-18_19.pdf	https://www.fec.gov/resources/cms-content/documents/notice1980-18-051380.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1980/1980-21.pdf	https://www.fec.gov/resources/cms-content/documents/notice1980-21-052380.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1980/45fr21209.pdf	https://www.fec.gov/resources/cms-content/documents/notice1980-040180.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1980/45fr43371.pdf	https://www.fec.gov/resources/cms-content/documents/notice1980-062780.pdf redirect;
+
+# /law/cfr/ej_compilation/1979/ redirects 
+rewrite ^/law/cfr/ej_compilation/1979/1979-20.pdf	https://www.fec.gov/resources/cms-content/documents/notice_1979-20_11051979.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1979/1979-23.pdf	https://www.fec.gov/resources/cms-content/documents/notice1979-23_12271979.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1979/44fr20336..pdf	https://www.fec.gov/resources/cms-content/documents/notice1979-040479.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1979/44fr33368.pdf	https://www.fec.gov/resources/cms-content/documents/notice_06081979.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1979/44fr339348.pdf	https://www.fec.gov/resources/cms-content/documents/notice1979-070579.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1979/44fr39350.pdf	https://www.fec.gov/resources/cms-content/documents/notice1979-070579.pdf redirect;
+rewrite ^/law/cfr/ej_compilation/1979/44fr63036.pdf	https://www.fec.gov/resources/cms-content/documents/notice_1979-11-01.pdf redirect;
+
+# /law/cfr/ej_compilation/1977/ redirects 
+rewrite ^/law/cfr/ej_compilation/1977/95-44.pdf https://www.fec.gov/resources/cms-content/documents/95-44.pdf redirect;
```

Comment:
Could you check the spacing between the redirect URLs here. Most seem to have multiple spaces.